packages/studio-plugin/src/messaging/registerGetEntityTypes.ts

```diff
@@ -0,0 +1,28 @@
+import { registerListener } from "./registerListener";
+import { ViteDevServer } from "vite";
+import { MessageID, ResponseType } from "../types";
+import ManagementApiService from "../http/ManagementApiService";
+
+/**
+ * Registers a listener for getting all enabled entity types from the account.
+ */
+export default function registerGetEntityTypes(
+  server: ViteDevServer,
+  managementApiService: ManagementApiService
+) {
+  registerListener(server, MessageID.GetEntityTypes, async () => {
+    try {
+      const response = await managementApiService.getEnabledEntityTypes();
+      return {
+        msg: "Successfully fetched entity types.",
+        type: ResponseType.Success,
+        entityTypes: response,
+      };
+    } catch {
+      return {
+        msg: "Error fetching entity types.",
```

Comment:
Do we want to include the original error in some way?

Reply:
I didn't think it was needed because these requests are purely internal. The user doesn't really know that we're making them. At most, I was thinking we'd display a generic error/warning message if they open the stream scope modal and we don't have some information populated from their account because we got some error. I don't think we'd take any specific action based on particular error messages, and the user likely wouldn't be able to either

Reply:
Maybe we would want to console error in the terminal? It doesn't need to be something the user sees unless they look for it

Reply:
Updated!